session/session.go

```diff
@@ -2257,6 +2250,7 @@ func (s *session) InitTxnWithStartTS(startTS uint64) error {
 	if err != nil {
 		return err
 	}
+	s.initTxn()
```

Comment:
Will this bring some side effects?

Reply:
PTAL @tiancaiamao , thx!

Reply:
To lower the risk, changes to `NewTxn` and `initTxnWithStartTS` are removed.